stress/v2/ponyExpress/ponyExpress_write_test.go

```diff
@@ -0,0 +1,84 @@
+package ponyExpress
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
```

Comment:
the only reason I ask is because this is the test that is failing on the circle build.